ecs.tf

```diff
@@ -198,11 +199,24 @@ resource "aws_iam_role_policy" "secretsRead_policy" {
   policy = data.aws_iam_policy_document.secretsRead_policy_document.json
 }
 
+data "aws_kms_alias" "ssm_key" {
+  count = length(var.kms_key_aliases)
```

Comment:
what's this `count`? I cannot find it [in the docs](https://www.terraform.io/docs/providers/aws/r/kms_alias.html). I thought that the alias was just for one key

Reply:
https://www.terraform.io/docs/configuration/resources.html#count-multiple-resource-instances-by-count

I learnt about this the other day!

Reply:
that's interesting!